### ISSUE TITLE
1.13.1 and 1.13.2 fail

### ISSUE THREAD

**Issue by martmists** [barneygale/quarry#58]
This is due to their versions and protocols not working in `quarry.data.packets`

```
  ...
    return packets.packet_idents[key]
KeyError: (404, 'status', 'downstream', 'status_response')
```

Despite these files being here they fail to load.
even `print(packets.minecraft_versions)` returns the following incorrect data:
`{210: '1.10', 335: '1.12', 110: '1.9.4', 107: '1.9', 315: '1.11', 340: '1.12.2', 4: '1.7.5', 109: '1.9.2', 108: '1.9.1', 338: '1.12.1', 47: '1.8.8', 393: '1.13', 316: '1.11.2'}`

**Comment by williammck** [comment]
1.13.1 and 1.13.2 support won't be available until the next release (likely v1.2).

In the meantime, you can update to what's in the master branch with `pip install --upgrade git+https://github.com/barneygale/quarry.git`.

**Comment by barneygale** [comment]
Pushed version 1.2 to pypi with support for 1.13.2. An upcoming release will support 1.14.
